build-support/bin/generate_github_workflows.py

```diff
@@ -400,6 +406,7 @@ def build_wheels_step(*, is_macos: bool) -> Step:
                     "name": "Build wheels and fs_util (Linux)",
                     "runs-on": LINUX_VERSION,
                     "container": "quay.io/pypa/manylinux2014_x86_64:latest",
+                    "timeout-minutes": 40,
```

Comment:
Should probably be 60, maybe 70. These take a long time, and even longer on branch builds than PRs because we build in release mode rather than debug